beacon-chain/rpc/prysm/v1alpha1/beacon/validators.go

```diff
@@ -659,153 +660,14 @@ func (bs *Server) GetValidatorPerformance(
 	ctx context.Context, req *ethpb.ValidatorPerformanceRequest,
 ) (*ethpb.ValidatorPerformanceResponse, error) {
 	if bs.SyncChecker.Syncing() {
-		return nil, status.Errorf(codes.Unavailable, "Syncing to latest head, not ready to respond")
-	}
-
-	headState, err := bs.HeadFetcher.HeadState(ctx)
-	if err != nil {
-		return nil, status.Errorf(codes.Internal, "Could not get head state: %v", err)
+		return nil, status.Error(core.ErrorReasonToGRPC(core.Unavailable), "syncing to latest head, not ready to respond")
```

Comment:
Here you can use `codes.Unavailable` directly, without calling the helper function